docs/gallery.yml

```
@@ -125,3 +125,6 @@
   repository: 
   image: https://raw.githubusercontent.com/datapartnership/welcome/master/images/logo.png
   website: https://docs.datapartnership.org
+- name: "The ISA Cookbook"
+  image: https://isa-tools.org/isa-api/_static/isa_footer_logo.svg
+  website: https://isa-tools.org/isa-api/content/index.html
```

```suggestion
  website: https://isa-tools.org/isa-api/content/index.html
  repository:
```

has to be present, even if it is null